opentech/apply/users/piplines.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+
+
+def make_otf_staff(backend, user, response, *args, **kwargs):
+    _, email_domain = user.email.split('@')
+    if email_domain in settings.STAFF_EMAIL_DOMAINS:
+        user.is_staff = True
```

Comment:
`is_staff` grants users access to the Django admin. Do we want that? I see 2 alternatives:
1. flag on the user model `is_apply_staff` (to abstract it a bit. `is_otf_staff` works too)
2. add the users to the "Staff" group